### PR TITLE
Add python-pip as package dependency for archlinux

### PR DESCRIPTION
**Pull request by netcrop**
## Description

Due to "pip3 install" failed in script linux_install.sh. python-pip has been added as package dependency for archlinux.
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR
#7040
* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: util/linux_install.sh [hunk 1]
```diff
@@ -213,4 +214,4 @@ else
 fi
 
 # Global install tasks
-pip3 install -r ${util_dir}/../requirements.txt
+sudo pip3 install -r ${util_dir}/../requirements.txt
```
Comment (fauxpark) [inline util/linux_install.sh]: [What are the risks of running 'sudo pip'?](https://stackoverflow.com/questions/21055859/what-are-the-risks-of-running-sudo-pip)

This is not a good idea. A better solution might be to use `--user` explicitly. It seems that is only sometimes the default.

**Comment by netcrop** [comment]
It turns out all the python requirements already are included in archlinux official repositories. "pacman -S --needed python-appdirs python-argcomplete python-colorama" will solve the issue for archlinux.

File: util/linux_install.sh [hunk 2]
```diff
@@ -77,6 +77,10 @@ elif grep ID /etc/os-release | grep -q 'arch\|manjaro'; then
 		gcc \
 		git \
 		python \
+		python-pip \
+		python-appdirs \
+		python-argcomplete \
+		python-colorama \
```
Comment (fauxpark) [inline util/linux_install.sh]: This is still not ideal IMO. Now, when we need to update the list of Python packages to install, it has to be done in two places. As I said, you should try to use the `--user` flag for pip to get around the permissions problems.

**Comment by netcrop** [comment]
@fauxpark , I understand your point now. and using pip install --user as requirements manager for python in install_linux.sh.

**Pull request merged**